server/routes/statistics.js

```diff
@@ -585,73 +602,12 @@ module.exports = function (config) {
 					result.chartsResult   = chartsResult;
 					return getFilesCompleted(query).then(function (res) {
 						result.filesCompleted = res;
+						return deferred(true);
```

Comment:
You could return `query` here and in next line just chain `getApprovedCerts` call like this:

```javascript
    return query;
}).then(getApprovedCerts).then(function (res) {
```

One less block level. But not a deal breaker.